src/server/rest/CentralRestServerService.ts

```diff
@@ -84,6 +84,7 @@ class RequestMapper {
           [ServerAction.BILLING_SYNCHRONIZE_INVOICES]: BillingService.handleSynchronizeInvoices.bind(this),
           [ServerAction.BILLING_FORCE_SYNCHRONIZE_USER_INVOICES]: BillingService.handleForceSynchronizeUserInvoices.bind(this),
           [ServerAction.BILLING_CREATE_TRANSACTION_INVOICE]: BillingService.handleCreateTransactionInvoice.bind(this),
+          [ServerAction.BILLING_ATTACH_PAYMENT_METHOD]: BillingService.handleBillingAttachPaymentMethod.bind(this),
```

Comment:
You can also use the new RESTful API by adding BillingRouter.ts class in server/rest/v1/router/api
And declare the billing class in the GlobalRoute.ts
Check TenantRouter class, it should work the same.
So no need changes in that class anymore.

Reply:
You can remove the old ones.
The old REST interface will be replaced by the new RESTful one.